web/srv/handlers_test.go

```diff
@@ -7,8 +7,11 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/ghodss/yaml"
 	"github.com/julienschmidt/httprouter"
+	helpers "github.com/linkerd/linkerd2/cli/cmd"
```

Comment:
This still introduces a dependency from the `web/srv` package to the `cli/cmd`, which I'd like to avoid, even if it's only for tests. Can you instead move the `cli/cmd/test_helpers.go` file that you're adding to `pkg/profiles/test_helper.go`?

Also, I'd call to `test_helper.go` (singular), instead of `test_helpers.go` (plural). Right now this branch is introducing a new file called `cli/cmd/test_helpers.go`, and `cli/cmd/test_helper.go` already exists.

Reply:
ah yes, good point! thanks!